src/Components/Footer.jsx

```diff
@@ -1,18 +1,59 @@
+import React from "react";
+import { Grid, Box, Container } from "@material-ui/core";
+import { List, ListItem } from "@material-ui/core";
+import { Stack, Typography } from "@mui/material";
+import { Divider } from "@mui/material";
+
+import democracylabLogo from "../Assets/democracylab-logo.png";
+import openseattleLogo from "../Assets/openseattle-logo.png";
+
 const Footer = () => {
   return (
-    <footer className="footer-cont">
-      <div className="info-cont">
-        <a className="white-link" href="mailto:cvp@seamus.work">
-          <ul>Email Us</ul>
-        </a>
-        <a className="white-link" href="https://www.democracylab.org/about">
-          <ul>Democracy Lab</ul>
-        </a>
-        <a className="white-link" href="https://openseattle.org/about/">
-          <ul>Open Seattle</ul>
-        </a>
-      </div>
-    </footer>
+    <Container maxWidth="xl">
+      <Stack direction="row" sx={{ display: "flex", padding: 5 }}>
+        <Grid container columns={2} spacing={6} justifyContent="space-between">
+          <Grid item>
+            <List>
+              <ListItem>
+                <a href="/" style={{ color: "#000000" }}>
```

Comment:
Let's do something similar and use `useStyles` and `makeStyles` to create a `linkStyles` class. Similar to what you do in the `TeamCard` component.

Reply:
So that we can replace each `style={{ color: "#000000"}}` with `className={classes.linkStyles}`

Reply:
Got it changed over.